tui-prompts/src/text_prompt.rs

```diff
@@ -88,7 +89,7 @@ impl<'a> StatefulWidget for TextPrompt<'a> {
         let width = area.width as usize;
         let height = area.height as usize;
         let value = self.render_style.render(state);
-        let value_length = value.chars().count();
+        let value_length: usize = value.width_cjk();
```

Comment:
```suggestion
        let value_width = value.width_cjk();
```

Also, this is only used once below, so perhaps just inline it?